frontend/src/stores/redux/user/slice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import {RootState} from "../store";
+import {ILoggedUserState} from "./types";
+import {UserType} from "./loggedUser";
+
+const initialState: ILoggedUserState = {
+    loggedUserState: {
+        id: 0,
+        email: "user@gmail.com",
+        username: "User",
+        userType: UserType.Customer,
+        token: ""
+    }
+}
+
+export const userSlice = createSlice({
```

Comment:
You name again inconsistently your files, vars and so on. If you name it as loggedUser, keep it everywhere. Because model user is different, it doesn't have token